From 8a5a298c0044d79ccc4649b626451a731de2ae82 Mon Sep 17 00:00:00 2001
From: linjh <linjh@rock-chips.com>
Date: Wed, 10 Oct 2012 09:59:31 +0800
Subject: [PATCH] rk2928-phonepad-sdk: add hi704 camera support

[reference files]
	modified:
		arch/arm/configs/rk2928_phonepad_defconfig
		arch/arm/mach-rk2928/board-rk2928-phonepad-camera.c
---
 arch/arm/configs/rk2928_phonepad_defconfig          | 3 +--
 arch/arm/mach-rk2928/board-rk2928-phonepad-camera.c | 4 ++--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/rk2928_phonepad_defconfig b/arch/arm/configs/rk2928_phonepad_defconfig
index e9893dbad88a..2ccaed60cad9 100644
--- a/arch/arm/configs/rk2928_phonepad_defconfig
+++ b/arch/arm/configs/rk2928_phonepad_defconfig
@@ -273,8 +273,7 @@ CONFIG_RK30_PWM_REGULATOR=y
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_VIDEO_DEV=y
 CONFIG_SOC_CAMERA=y
-CONFIG_SOC_CAMERA_GC0308=y
-CONFIG_GC0308_USER_DEFINED_SERIES=y
+CONFIG_SOC_CAMERA_HI704=y
 CONFIG_VIDEO_RK29=y
 CONFIG_VIDEO_RK29_CAMMEM_ION=y
 CONFIG_ION=y
diff --git a/arch/arm/mach-rk2928/board-rk2928-phonepad-camera.c b/arch/arm/mach-rk2928/board-rk2928-phonepad-camera.c
index ccb65c2195c5..496cc524180c 100755
--- a/arch/arm/mach-rk2928/board-rk2928-phonepad-camera.c
+++ b/arch/arm/mach-rk2928/board-rk2928-phonepad-camera.c
@@ -70,8 +70,8 @@
 #define CONFIG_SENSOR_SVGA_FPS_FIXED_02      15000
 #define CONFIG_SENSOR_720P_FPS_FIXED_02      30000
 
-#define CONFIG_SENSOR_1 RK29_CAM_SENSOR_GC0308                      /* front camera sensor 0 */
-#define CONFIG_SENSOR_IIC_ADDR_1 	   0x42
+#define CONFIG_SENSOR_1 RK29_CAM_SENSOR_HI704                      /* front camera sensor 0 */
+#define CONFIG_SENSOR_IIC_ADDR_1 	   0x60
 #define CONFIG_SENSOR_IIC_ADAPTER_ID_1	  1
 #define CONFIG_SENSOR_CIF_INDEX_1				  0
 #define CONFIG_SENSOR_ORIENTATION_1       270
-- 
2.35.3

